client/landing/stepper/declarative-flow/internals/steps-repository/migration-upgrade-plan/index.tsx

```diff
@@ -1,44 +1,30 @@
 import { StepNavigationLink } from '@automattic/onboarding';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
-import { FC } from 'react';
 import { Step } from '../../types';
 import SiteMigrationUpgradePlan from '../site-migration-upgrade-plan';
 import { MigrationUpgradePlanActions as Actions } from './actions';
 
-interface Props {
-	onClick: () => void;
-	label: string;
-}
-const ImportButton: FC< Props > = ( { onClick, label } ) => {
-	return (
-		<div className="step-container__skip-wrapper">
-			<StepNavigationLink
-				direction="forward"
-				handleClick={ onClick }
-				label={ label }
-				cssClass={ clsx( 'step-container__navigation-link', 'has-underline' ) }
-				borderless
-			/>
-		</div>
-	);
-};
-
 const MigrationUpgradePlan: Step = ( props ) => {
 	const translate = useTranslate();
 	const { navigation } = props;
+
+	const handleClick = () =>
+		navigation?.submit?.( {
+			action: Actions.IMPORT_CONTENT_ONLY,
+		} );
+
 	return (
 		<SiteMigrationUpgradePlan
 			{ ...props }
 			headerText={ translate( 'Here’s the plan you need' ) }
 			customizedActionButtons={
-				<ImportButton
+				<StepNavigationLink
```

Comment:
It's just a small refactor. I thought to extract this to another component since we'd need to re-use it, but I thought it would be simpler to just call the component directly since the wrapper is not needed. It also fixes the className.